### ISSUE TITLE
Window of MuditaCenter goes blank and unresponsive

### ISSUE THREAD

**Issue by esope** [mudita/mudita-center#1612]
# 🐛 Bug report

## 📝 Description

A few seconds after startup, the window of MuditaCenter goes blank and does not respond.
This makes MuditaCenter unusable: backups and upgrades are impossible to make, for example.

The following lines show up in the log:

```
Error sending from webFrameMain:  Error: Render frame was disposed before WebFrameMain could be accessed
    at WebFrameMain.send (node:electron/js2c/browser_init:2:94617)
    at WebContents.send (node:electron/js2c/browser_init:2:79767)
    at c.sendToRenderers (/tmp/.mount_MuditaGKp6dz/resources/app.asar/dist/main.js:14:65979)
    at /tmp/.mount_MuditaGKp6dz/resources/app.asar/dist/main.js:8:2948353
```

## 👍 Expected behavior

This should not happen.

## 👎 Current behavior

MuditaCenter cannot be used.

## 🔬 Minimal Reproduction

Launch MuditaCenter. The fact that a Pure is plugged or not does not change the outcome.
Wait a few seconds. The window goes blank and does not answer to keyboard/mouse interactions.

## 🌍 Your operating system and Mudita Center version

MuditaCenter 2.2.7
OS: LinuxMint 21.2, linux kernel 6.5.0

## 🤔 Anything else relevant?

Nothing.

**Comment by esope** [comment]
Is anybody looking at this issue ? Please ?

**Comment by esope** [comment]
Same issue with version 2.2.8 of Mudita Center.
Because of this, Mudita Center cannot be used at all.
PLEASE DO HAVE A LOOK AT THIS ISSUE

The error message in the log is the same.

```
Error sending from webFrameMain:  Error: Render frame was disposed before WebFrameMain could be accessed
    at WebFrameMain.send (node:electron/js2c/browser_init:2:94617)
    at WebContents.send (node:electron/js2c/browser_init:2:79767)
    at c.sendToRenderers (/tmp/.mount_Mudita0Ff6xH/resources/app.asar/dist/main.js:14:65979)
    at /tmp/.mount_Mudita0Ff6xH/resources/app.asar/dist/main.js:8:2947663
```

According to some forums, it might be an issue with the version of Electron.

**Comment by tomaszmaleckimudita** [comment]
Dear @esope. 

Some users had been able to resolve the issue by adding user to dialout group responsible for USB access to devices. 

Could you first check if your ubuntu user has access to `dialout` group? 
`groups $USER`

If not, run a command: `sudo gpasswd -a $USER dialout` and check if this solved your problem. 

If that didn't resolve your issues - we are working on finding different solution for this issue for remaining users. We are currently reproduced the issue on two of our machines. The suspected cause of the issue is hardware acceleration from Intel GPU not correctly supported by Electron.

**Comment by esope** [comment]
Thanks @tomaszmaleckimudita. I am part of the `dialout` group.

This issue is not related to access rights of USB devices.

More likely, this is a concurrency issue either in the code of Mudita center, or in the version of Electron that is included in this version of Mudita center.

**Comment by IlyaBizyaev** [comment]
I have the same problem.

Operating System: openSUSE Tumbleweed 20240328
KDE Plasma Version: 6.0.80
KDE Frameworks Version: 6.1.0
Qt Version: 6.6.3
Kernel Version: 6.8.1-1-default (64-bit)
Graphics Platform: Wayland
Graphics Processor: NVIDIA GeForce RTX 4070/PCIe/SSE2

**Comment by esope** [comment]
Fixed with version 2.3.0. Thanks!
